app/views/layouts/my_facilities.html.erb

```diff
@@ -95,20 +100,25 @@
     <div class="col-lg-2 pl-lg-0 desktop">
       <nav class="secondary-nav">
         <%= link_to 'Overview', my_facilities_overview_path,
-                    class: request.path == my_facilities_overview_path ? "nav-link active" : "nav-link" %>
+                    class: "nav-link #{active_action?("index")}" %>
         <%= link_to 'BP control', my_facilities_blood_pressure_control_path(preserve_query_params(request.query_parameters, ["size"])),
-                    class: request.path == my_facilities_blood_pressure_control_path ? "nav-link active" : "nav-link" %>
+                    class: "nav-link #{active_action?("blood_pressure_control")}" %>
         <%= link_to 'Registrations', my_facilities_registrations_path(preserve_query_params(request.query_parameters, ["size", "period"])),
-                    class: request.path == my_facilities_registrations_path ? "nav-link active" : "nav-link" %>
+                    class: "nav-link #{active_action?("registrations")}" %>
+        <%= link_to 'Missed Visits', my_facilities_missed_visits_path(preserve_query_params(request.query_parameters, ["size", "period"])),
+                    class: "nav-link #{active_action?("missed_visits")}" %>
       </nav>
       <% unless request.path == my_facilities_overview_path %>
         <section id="facility-filters" class="secondary-nav-filters">
           <h4>Facility Filters</h4>
           <div class="dropdown show">
             <a class="btn btn-sm dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-              All blocks
+              <% if @selected_zones.size == 1 %>
+                <%= @selected_zones.first&.capitalize %>
```

Comment:
If we have one selected zone, why do we need to use the safe navigation `&.capitalize`?

Reply:
removing the `&`, think it got left behind from a refactor